libscpi/test/test_parser.c

```diff
@@ -78,6 +78,7 @@ static const scpi_command_t scpi_commands[] = {
     { .pattern = "SYSTem:VERSion?", .callback = SCPI_SystemVersionQ,},
 
     { .pattern = "STATus:QUEStionable[:EVENt]?", .callback = SCPI_StatusQuestionableEventQ,},
+    { .pattern = "STATus:QUEStionable:CONDition?", .callback = SCPI_Status_QuestionableConditionQ,},
```

Comment:
You have typo in function name `SCPI_Status_QuestionableConditionQ` -> `SCPI_StatusQuestionableConditionQ`. See [travis build](https://travis-ci.org/j123b567/scpi-parser/builds/309262791#L1511)